analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -73,7 +73,7 @@ object VirtualEnv : CommandLineTool {
     override fun command(workingDir: File?) = "virtualenv"
 
     // Allow to use versions that are known to work. Note that virtualenv bundles a version of pip.
-    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[15.1,16.1]")
+    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[15.1.0,16.4.3]")
```

Comment:
Should we use "[15.1.0,16.5[" to allow patch level upgrades?

Reply:
Yes, done.